crates/libmoonlight/src/installer.rs

```diff
@@ -1,13 +1,8 @@
 use crate::{get_app_dir, get_moonlight_dir, PATCHED_ASAR};
-
-use super::{
-    types::*,
-    util::{get_branch_config, get_download_dir},
-};
+use super::{types::*, util::get_download_dir};
 use std::path::PathBuf;
 
-const USER_AGENT: &str =
-    "moonlight-installer (https://github.com/moonlight-mod/moonlight-installer)";
+const USER_AGENT: &str = "moonlight-installer (https://github.com/moonlight-mod/moonlight-installer)";
```

Comment:
Your `cargo fmt` seems to have different line widths than the default - could you reformat? (This affects a few other files as well)